astropy/convolution/convolve.py

```diff
@@ -312,6 +312,9 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0, crop=True,
         256.
     quiet : bool, optional
         Silence warning message about NaN interpolation
+    normalized_epsilon: float, optional
+        The upper limit on the deviation of the kernel sum with respect to unity.
+        Will be checked only if normalize_kernel is False.
```

Comment:
Also if a parameter is optional it is custom to provide or hint at the default value. Something like `Default is "1e-8".` (or defaults to xxx, ...)

Another thing that could be added is a sphinx directive when the parameter is added, something like: `.. versionadded:: 1.3` (or 1.2.x).

This reminds me: This PR needs a Changelog entry (home directory `Changes.rst`) for the appropriate version/subpackage under "API changes" (or "New features").